scripts/background.js

```diff
@@ -424,16 +429,13 @@ chrome.runtime.onMessage.addListener(function(message,sender,sendResponse){
                     alert("Structure as radial tree not available on archive.org pages");
                 }else{
                     chrome.tabs.executeScript(tab.id, {
-                      file:"scripts/d3.js"
-                    });
-                    chrome.tabs.executeScript(tab.id, {
-                      file:"scripts/radial-tree.umd.js"
+                            file:"scripts/d3.js"
```

Comment:
Please don't change the indentation of these lines.

Reply:
Sorry @vbanos. There is another mistake in it. I will fix them